banklib/src/lib.rs

```diff
@@ -0,0 +1,299 @@
+use std::io::{Read, Write};
+use std::net::TcpStream;
+
+use serde::{Deserialize, Serialize};
+
+// const SERVER_ADDRESS: &str = "127.0.0.1:7878";
+
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+pub enum Command {
+    CreateAccount(String),
+    IncreaseAccount(String, u32),
+    DecreaseAccount(String, u32),
+    Transfer(String, String, u32),
```

Comment:
На будущее предлагаю писать вот так:

```suggestion
    Transfer {
        from: String,
        to: String,
        amount: u32,
    }
```

Чтобы отправителя и получателя было не так просто перепутать местами